src/main/webapp/view/transcript/transcript.js

```diff
@@ -171,3 +191,12 @@ function scrollToTopOfTranscript(transcriptLine) {
   const ulElementOffset = transcriptLine.parentElement.offsetTop;
   transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
 }
+
+/**
+ * Resets the transcript container to what it looks like before it
+ * started scrolling.
+ */
+function resetToBeforeTranscript() {
```

Comment:
Is this used right now? It doesn't do anything so maybe add it when you need it

Reply:
Not really, but I added a function call for this in `seekTranscript` where I would like to call it once it is implemented. I thought of adding a stub as like a TODO. I'll keep it here if that's okay.

Reply:
Ah okay